backend/app/notify/notify.go

```diff
@@ -35,8 +35,16 @@ type Store interface {
 }
 
 type Request struct {
```

Comment:
I have no better ideas how to include the extra info you need into request, but generally it bothers me a little bit. This struct became too specialized for email support. I'm mot saying this is bad, just a feeling.

Reply:
Addressed by a2281931edaa333d884c720dab736d051715df29, now it is still ugly but at least have a decent description of what is expected and what purpose Request does serve.

Please see if it that's better than was before.

Reply:
well, it doesn't change much, but extra comments won't hurt for sure